src/modules/site-v2/base/static/js/plot.js

```diff
@@ -161,7 +161,7 @@ function render_scatterplot_histograms(container_selector, data, config={}) {
     const height = (config['height'] || 400) - margin.top - margin.bottom;
 
     // Read values from config, filling in default values when not supplied
-    const circle_radius = config['circle_radius'] || 4;
+    const circle_radius = config['circle_radius'] || 8;
```

Comment:
consider using `??` for default values instead of `||` -- this avoids an edge case where the actual value retrieved is zero (Valid), but `||` jumps to the default value instead.  

Example `||`:
```
config['circle_radius'] = undefined; // this value is not defined.
const circle_radius = config['circle_radius'] || 8;  // will yield 8, which is correct.

config['circle_radius'] = 0;  
const circle_radius = config['circle_radius'] || 8;  // will yield 8 (!) instead of zero, which is not correct.
```

Alternative `??`:
```
config['circle_radius'] = undefined; // the value for the key `circle_radius` in config is not set (undefined)
const circle_radius = config['circle_radius'] ?? 8;  // will yield 8, which is correct.

config['circle_radius'] = 0; 
const circle_radius = config['circle_radius'] ?? 8;  // will yield 0, which is correct
```